jraft-core/src/main/java/com/alipay/sofa/jraft/core/NodeImpl.java

```diff
@@ -2861,6 +2867,28 @@ public UserLog readCommittedUserLog(final long index) {
         throw new LogNotFoundException("User log is deleted at index: " + curIndex);
     }
 
+    @Override
+    public void registerReplicatorStateListener(final Replicator.ReplicatorStateListener replicatorStateListener) {
+        Requires.requireNonNull(replicatorStateListener, "ReplicatorStateListener");
```

Comment:
"ReplicatorStateListener" -> "replicatorStateListener"

Reply:
Okay,I will fix this codes comments and then push it again.Thanks!